src/components/Tool/Appeal/AppealDetails/AppealLeftPanel/AppealsLeftPanel.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { DynamicFilterPanel } from 'src/components/Shared/Filters/DynamicFilterPanel';
+import { ContextTypesEnum } from 'src/components/Shared/Filters/FilterPanel';
+import { TableViewModeEnum } from 'src/components/Shared/Header/ListHeader';
+import {
+  AppealsContext,
+  AppealsType,
+} from '../../AppealsContext/AppealsContext';
+import { DynamicAppealsListFilterPanel } from '../../List/AppealsListFilterPanel/DynamicAppealsListFilterPanel';
+
+export const AppealsLeftPanel: React.FC = () => {
+  const {
+    filterData,
+    filtersLoading,
+    savedFilters,
+    activeFilters,
+    toggleFilterPanel,
+    setActiveFilters,
+    viewMode,
+  } = React.useContext(AppealsContext) as AppealsType;
+
+  return viewMode !== TableViewModeEnum.Flows ? (
+    <DynamicAppealsListFilterPanel onClose={toggleFilterPanel} />
```

Comment:
When a user clicks on an Appeal, could we have this filter panel open automatically for list view? I most commonly click an appeal and then I want to go directly to either the 'Given' category or the 'Committed' category. Usually on angular the panel is open by default.

Reply:
Also, when clicking an appeal it should automatically show the contacts in the 'Asked' category.

Reply:
Done & Done. Good catches